.vscode/launch.json

```diff
@@ -336,6 +337,30 @@
                 "/Views": "${workspaceFolder}/Views"
             }
         },
+        {
+            "name": "Notifications-SelfHost",
+            "presentation": {
+                "hidden": true,
+                "group": "self-host",
+                "order": 999
+            },
+            "requireExactSource": true,
+            "type": "coreclr",
+            "request": "launch",
+            "preLaunchTask": "buildNotifications",
+            "program": "${workspaceFolder}/src/Notifications/bin/Debug/net6.0/Notifications.dll",
+            "args": [],
+            "cwd": "${workspaceFolder}/src/Notifications",
+            "stopAtEntry": false,
+            "env": {
+                "ASPNETCORE_ENVIRONMENT": "Development",
+                "ASPNETCORE_URLS": "http://localhost:61841",
+                "developSelfHosted": "true"
+            },
+            "sourceFileMap": {
+                "/Views": "${workspaceFolder}/Views"
+            }
+        },
```

Comment:
While you're here, can you add a non self-host Notifications block here for local debugging? 😄  
If I'm not mistaken, it would be the same but the group would be `cloud`, and it can be added to the `Full Server` configuration.

Reply:
I added Notifications and added it to the "Full Server" configuration.  Great catch!